### PR TITLE
CI

### PR DESCRIPTION
**Pull request by lukesneeringer**
Make CI great again!

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+          command: pip install codecov nox-automation
+      - run:
+          name: Run unit tests.
+          command: nox -s "unit_tests(python_version='3.4')"
+      - run:
+          name: Submit coverage data to codecov.
+          command: codecov
+    working_directory: /var/code/duck
+
+  node8:
```
Comment (jch-gh) [inline .circleci/config.yml]: You mean `python36:` right?

**Review by jch-gh — CHANGES_REQUESTED** [review]

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    docker:
+      - image: python:3.6
+    steps:
+      - checkout
+      - run:
+          name: Install nox and codecov.
+          command: pip install codecov nox-automation
+      - run:
+          name: Run unit tests.
+          command: nox -s "unit_tests(python_version='3.4')"
```
Comment (jchappell82) [inline .circleci/config.yml]: I think this is a version mismatch?

File: .circleci/config.yml [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+    docker:
+      - image: python:3.5
+    steps:
+      - checkout
+      - run:
+          name: Install nox and codecov.
+          command: pip install codecov nox-automation
+      - run:
+          name: Run unit tests.
+          command: nox -s "unit_tests(python_version='3.4')"
```
Comment (jchappell82) [inline .circleci/config.yml]: Version mismatch?

**Review by jch-gh — APPROVED** [review]
LGTM :shipit:

File: .circleci/config.yml (hunk 1 above)
Comment (lukesneeringer) [reply .circleci/config.yml]: Yep. Fixed already. :-)

**Pull request merged**